onnx/defs/nn/defs.cc

```diff
@@ -1229,4 +1225,84 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetDoc(LRN_ver1_doc)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
-} // namespace ONNX_NAMESPACE
+ONNX_OPERATOR_SET_SCHEMA(
+    ConvInteger,
+    8,
+    OpSchema()
+        .SetDoc(
+            "The integer convolution operator consumes an input tensor, a filter and a padding value, and computes the output. "
+            "The production MUST never overflow. The accumulation may overflow if and only if in 32 bits.")
+        .Input(
+            0,
+            "X",
+            "Input data tensor from previous layer; "
+            "has size (N x C x H x W), where N is the batch size, "
+            "C is the number of channels, and H and W are the "
+            "height and width. Note that this is for the 2D image. "
+            "Otherwise the size is (N x C x D1 x D2 ... x Dn)",
+            "T1")
+        .Input(
+            1,
+            "W",
+            "The weight tensor that will be used in the "
+            "convolutions; has size (M x C x kH x kW), where C "
+            "is the number of channels, and kH and kW are the "
+            "height and width of the kernel, and M is the number "
+            "of feature maps. For more than 2 dimensions, the "
+            "kernel shape will be (M x C x k1 x k2 x ... x kn), "
+            "where is the dimension of the kernel",
+            "T2")
+        .Input(
```

Comment:
what about bias?

Reply:
bias is kept to be separate as fusing it together with conv asks its scale and zero point to meet some equation. meanwhile, bias quantization is not that critical as input X and kernel W.

Reply:
so, no bias in convinteger here.

Reply:
@liwchang , if it's not check-able, then we should not add it. I assume that it will introduce ambiguity and correctness bugs.